packages/gensx/src/index.ts

```diff
@@ -41,6 +41,7 @@ export async function runCLI() {
     .description("Start a local GenSX server")
     .argument("<file>", "File to serve")
     .option("-p, --project <name>", "Project name")
+    .option("--environment <name>", "Environment name")
```

Comment:
@jmoseley - FYI, i didn't include a short hand version of `--environment` because `-e` was already taken and i figure this will almost always be set via the gensx.yaml file.

Curious if you have any opinions here though

Reply:
oh right. that will be confusing.... `-e` and `--env` in the same command. Should have called them stacks 🤣 

but `--env` seems fine.

Reply:
updated environment to be -e and --env and environment variables to be -ev --env-var